### ISSUE TITLE
Make font size changeable 

### ISSUE THREAD

**Issue by paulle** [3zpnix/WriteOn#11]
Please make font size changeable. This would be a little help for persons with bad eyes.
Thank you

**Comment by 3zpnix** [comment]
Hmm. I'll see what I can do. Thanks for suggesting ^^

**Comment by sphh** [comment]
Would it also be possible to reduce the space between lines in the overview? The large space helps with reading, but takes up a lot of (valuable) screen space …

**Comment by 3zpnix** [comment]
Yes.. that would be possible. I would have to rewrite some of the codes from a different area in order for it to not cause other bugs, but I can make it work. I'll put your suggestion in mind.

**Comment by 3zpnix** [comment]
Thank you for the suggestion. More updates coming soon!

**Comment by paulle** [comment]
Thank you

Gesendet von Proton Mail für Android

-------- Ursprüngliche Nachricht --------
Am 24.02.25 18:16 um 3zpnix schrieb :
